clients/clients-integration-tests/src/test/java/org/apache/kafka/clients/consumer/ShareConsumerTest.java

```diff
@@ -2994,35 +2992,4 @@ private void waitForAssignment(String groupId, List<TopicPartition> tps) {
             throw new RuntimeException(e);
         }
     }
-
-    public static class SerializerImpl implements Serializer<byte[]> {
```

Comment:
why moving it to `ClientsTestUtils`?

Reply:
I noticed that `BaseConsumerTest.scala` contains the same class, but it's only used in `PlaintextConsumerTest.scala`. So, when we rewrite `PlaintextConsumerTest.scala` in Java, we should move this class to a utility class. I will revert now.